apps/daimo-mobile/src/debugLog.ts

```diff
@@ -38,3 +50,77 @@ export function getDebugLog(headerLines: string[]) {
   const log = logs.join("\n") + `\n${now} - debug log captured`;
   return [`# Daimo Debug Log`, ...headerLines, log].join("\n\n");
 }
+
+/**
+ * Returns a function to send a complete debug log + env summary.
+ * Additionally returns the env summary for display.
+ */
+export function useSendDebugLog(
+  account?: Account | null
+): [() => Promise<void>, Record<string, string>] {
+  const [sec, setSec] = useState<Awaited<ReturnType<typeof getHardwareSec>>>();
+
+  useEffect(() => {
+    getHardwareSec().then(setSec);
+  }, []);
+
+  if (account == null) {
+    account = getAccountManager().currentAccount;
+  }
+  if (account == null) {
+    return [async () => {}, {}];
```

Comment:
might be misunderstanding something but wouldn't this mean it doesn't work during onboarding? would be nice if it did